sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-multi-database/src/main/java/com/azure/spring/sample/multi/database/MultiDatasourceApplication.java

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.sample.multi.database;
+
+import com.azure.cosmos.models.PartitionKey;
+import com.azure.spring.data.cosmos.core.CosmosTemplate;
+import com.azure.spring.data.cosmos.core.ReactiveCosmosTemplate;
+import com.azure.spring.data.cosmos.repository.support.CosmosEntityInformation;
+import com.azure.spring.sample.multi.database.database1.User;
+import com.azure.spring.sample.multi.database.database1.UserRepository;
+import com.azure.spring.sample.multi.database.database2.Book;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import java.util.List;
+
+@SpringBootApplication
+public class MultiDatasourceApplication implements CommandLineRunner {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private UserRepositoryForMYSQL userRepositoryForMYSQL;
+
+    @Autowired
+    @Qualifier("secondaryDatabaseTemplate")
+    private CosmosTemplate secondaryDatabaseTemplate;
+
+    @Autowired
+    @Qualifier("secondaryDatabase2Template")
+    private CosmosTemplate secondaryDatabase2Template;
+
+    @Autowired
+    @Qualifier("primaryDatabase2Template")
+    private ReactiveCosmosTemplate primaryDatabase2Template;
```

Comment:
Now we have 2 data-source, and 4 data-base:
 - primary-datasource
   + primary-database-1
   + primary-database-2
 - secondary-datasource
   + secondary-database-1
   + secondary-database-2

Can we just keep 2 data-source and 2 data-base? Like this:
 - primary-datasource
   + primary-database
 - secondary-datasource
   + secondary-database

Reply:
Azure-spring-data-cosmos supports multi-database configuration, including "multiple database accounts" and "single account, with multiple databases, do we need to consider only multiple database accounts? 
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-spring-data-cosmos/README.md#multi-database-configuration

Reply:
How about creating 2 samples: `azure-spring-boot-sample-cosmos-multiple-account` and `azure-spring-boot-sample-cosmos-single-account-multiple-database`. Write them in one project will be confusing.

Reply:
I got it